sbaid/common/simulator_type.py

```diff
@@ -0,0 +1,16 @@
+"""this module defines the SimulatorType class"""
+from gi.repository import GObject
+
+class SimulatorType(GObject.GObject):
+    """This class represents a type of simulator.
```

Comment:
Also ich kann mich dran erinnern, dass hier in der klassenbeschreibung wesentlich mehr stand. Wenn wir echt schon documentation schreiben, dann würd ich sie direkt richtig schreiben. Ansonsten einfach ein grosses TODO hinmachen (was ich vllt besser fände ig weils einfach schneller geht und wir ja echt nur das skeleton wollen). Weil dann kann man das halt machen wenn man die richtig macht vor allem bei methoden ist das dann halt besser weil man den ganzen kontext hat von dem was man schreibt. Sonst muss man alles doppelt machen und das ist immer irgendwie blöd oder